src/git/printable.go

```diff
@@ -10,6 +10,22 @@ import (
 
 var noneString = "[none]"
 
+// getBranchAncestryRoots returns the branches with children and no parents
+func getBranchAncestryRoots() []string {
```

Comment:
If you want to unit-test logic like this, we have pretty good infrastructure for this. An example is https://github.com/git-town/git-town/blob/master/test/git_repository_test.go#L324-L342. `createTestGitTownRepo` creates a fully initialized Repo for you.

Reply:
As time has passed going to skip this as feature test covers it well enough. Can add unit tests down the line if desired.